### DISCUSSION TITLE
update User Guide for DIY solution

### DISCUSSION THREAD

**Discussion by Ashley72** [Azure/Enterprise-Scale#825] [Q&A] [answered]
Could the instructions for the Do-It-Yourself deployment instructions for Enterprise-Scale using Azure PowerShell be updated to explain what the templateFile value should be and where is best place to run this, Azure Cloud Shell of locally? also what folder should PS1 script be run once repo cloned.

**Comment by krnese** [comment, marked as answer]
Can you see if this documentation helps? https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/deploy-powershell

We may add a blurb/link to this document from the DIY guidance, if you think that would be helpful.
